src/traces/parcats/constants.js

```diff
@@ -0,0 +1,18 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+
+module.exports = {
+    maxDimensionCount: 12,
```

Comment:
Is this a performance limit, a screen-space limit, a code structure limit, or... ?

Reply:
It's not performance or code structure. I think it was a conservative estimate based on plot width.  The rectangle width plus space for a single letter label over 12 dimensions, plus the left and right margins, comes to ~400 pixels.

Reply:
I'd remove this limitation then. `parcoords` has a hard limit at 60 dimensions based on the data structures we create on the GPU. We don't want to restrict plots just because we think they're not going to look good, or fit in a normal amount of space. Someone will surely want to fill up their dual 5K displays with these at some point ;)

Reply:
Done in https://github.com/jonmmease/plotly.js/pull/1/commits/6797a8354bcab0277af4e7d4cca9238bb19e415f